orttraining/orttraining/training_ops/cuda/optimizer/clip_grad_norm/clip_grad_norm.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <string>
+#include "core/common/common.h"
+#include "core/providers/cuda/cuda_kernel.h"
+
+namespace onnxruntime {
+namespace cuda {
+
+class ClipGradNorm final : public CudaKernel {
+ public:
+  ClipGradNorm(const OpKernelInfo& info) : CudaKernel(info) {
+    info.GetAttrOrDefault("max_norm", &max_norm_, 1.0f);
+    info.GetAttrOrDefault("norm_type", &norm_type, std::string("fro"));
```

Comment:
is this attribute used currently?

Reply:
Currently, this attribute is unused. We only perform the ReduceAllL2 (`frobenius` norm) for now. But we can potentially add other norm_types in the future.

Reply:
If so, and considering all subsquent compute logic is limited to fro algo, should we just add a check here for norm_type.

Reply:
I added a ENFORCE check in the cc file. Let me move it here.